framework/content/ecl-typography/ecl-typography-headings/README.md

```diff
@@ -1,4 +1,4 @@
 # Headings
```

Comment:
Please add the docs from wiki https://webgate.ec.europa.eu/CITnet/confluence/pages/viewpage.action?pageId=650250290#Typography..-Headlinesandheadings